lib/svgeez/destination.rb

```diff
@@ -0,0 +1,41 @@
+module Svgeez
+  class Destination
+    DEFAULT_DESTINATION_FILE_NAME = 'svgeez.svg'.freeze
+
+    def initialize(options = {})
+      @options = options
+    end
+
+    def file_id
+      @file_id ||= File.basename(file_name, '.svg').tr(' ', '-')
+    end
+
+    def file_name
+      @file_name ||=
+        if destination.end_with?('.svg')
+          File.split(destination)[1]
+        else
+          DEFAULT_DESTINATION_FILE_NAME
+        end
```

Comment:
This method (and `folder_path` method below) feel… slightly off. `destination.end_with?('.svg')` is repeated and the conditionals feel a bit clunky.